### ISSUE TITLE
Bug correction on SockJS haven't solved the problem

### ISSUE THREAD

**Issue by ArturFortunato** [blackhorse-one/stomp_dart_client#37]
Hello.

I'm trying to connect to my websocket server like this
```
stompClient = StompClient(
            config: StompConfig.SockJS(
                url: 'https://<server>/ws-connect',
                onConnect: onConnect,
                onStompError: (frame) => print("ERROR")
            )
        );

        stompClient.activate();
```

My Spring server is configured like this:

```
@Configuration
@EnableWebSocketMessageBroker
public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {

	@Override
	public void configureMessageBroker(MessageBrokerRegistry config) {
		config.enableSimpleBroker("/ws-events");
		config.setApplicationDestinationPrefixes("/app");
	}

	@Override
	public void registerStompEndpoints(StompEndpointRegistry registry) {
		registry.addEndpoint("/ws-connect")
		.setAllowedOrigins("*")
		.withSockJS();
	}
}
```
But the onconnect function never runs, neither does the onStompError. 
I saw that there was a bug that was already corrected on #33, but I already changed my pubspec.yaml file like this 

```
stomp_dart_client:
    git:
      https://github.com/blackhorse-one/stomp_dart_client.git
```

I know the server is working correctly, because I created a small javascript client like this:

```
var socket = new SockJS('https://<server>/ws-connect');

    stompClient = Stomp.over(socket);
    stompClient.connect({}, function (frame) {
        setConnected(true);
        console.log('Connected: ' + frame);
        stompClient.subscribe('/ws-events/' + $("#roomID").val(), function (greeting) {
            showGreeting(JSON.parse(greeting.body).content);
        });
    });
```
 and this works perfectly. 
Am I doing anything wrong?

**Comment by ArturFortunato** [comment]
The problem was with the certificate, but I'll still leave the issue opened, as an exception should be thrown when the certificate is invalid. I had to dig deep into the library to understand the problem

**Comment by KammererTob** [comment]
I am a bit busy these days but i will have a look soon

**Comment by KammererTob** [comment]
Hey. I've just released 0.4.0, which hopefully should forward all errors to the `onWebSocketError` callback. If this does not fix your issue, please feel free to re-open it.
